pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Topics.java

```diff
@@ -849,6 +850,62 @@ default TopicStats getStats(String topic) throws PulsarAdminException {
         return getStatsAsync(topic, false, false);
     }
 
+
+    /**
+     * Get the stats for non-persistent topic.
+     * All the rates are computed over a 1 minute window and are relative the last completed 1 minute period.
+     *
+     * @param topic
+     *            topic name
+     * @param getPreciseBacklog
+     *            Set to true to get precise backlog, Otherwise get imprecise backlog.
+     * @param subscriptionBacklogSize
+     *            Whether to get backlog size for each subscription.
+     * @return the topic statistics
+     *
+     * @throws NotAuthorizedException
+     *             Don't have admin permission
+     * @throws NotFoundException
+     *             Topic does not exist
+     * @throws PulsarAdminException
+     *             Unexpected error
+     */
+    NonPersistentTopicStats getStatsNonPersistent(String topic, boolean getPreciseBacklog,
```

Comment:
The NonPersistentTopicStats also implement the TopicStats, So I think we do not need to introduce a new method in the API? We can just use the `getStats` method, and if users want to get the msgDropRate, they can convert TopicStats to NonPersistentTopicStats directly?

Reply:
@codelipenghui 
I've tried that before, once the response object got serialized as TopicStats, even we convert it to NonPersistentTopicStats later the msgDropRate field will be missing, I think it's just lost during deserialization, so seems we'll have to deserialize to the desired type or interface when we get the response(NonPersistentTopicStats).

Reply:
@codelipenghui 
I just tried that, once the response object got deserialized as TopicStats, it's actual type will be TopicStatsImpl at client side(we set the binding at ObjectMapperFactory [ref](https://github.com/apache/pulsar/blob/3cbfd7454ce824a720d1dc645ea23ee3831ffff0/pulsar-common/src/main/java/org/apache/pulsar/common/util/ObjectMapperFactory.java#L179)), even server send it out as NonPersistentTopicStatsImpl, and we won't be able to convert the object back to NonPersistentTopicStats or NonPersistentTopicStatsImpl. 
I think we'll have to deserialize the response to the desired type when we receive it. So a new method is needed for this purpose.